src/man/sssd.conf.5.xml

```diff
@@ -1001,6 +1001,11 @@ fallback_homedir = /home/%u
                             Default: True
                         </para>
                         <para>
+                            WARNING: Setting this option to false will have
+                            very negative impact on performance. Using this
+                            option is highly discouraged.
+                        </para>
```

Comment:
@mzidek-rh, what about:
"WARNING: Using this option is strongly discouraged. It's been designed mainly for testing purposes as setting it to false has a very negative impact on performance".

The text above is just a suggestion, yours look good as it is IMO. I'd like to hear from other developers on this as well.

Reply:
Also, I'd prefer to have both patches squased.